src/encoded/schemas/changelogs/biosample.md

```diff
@@ -4,7 +4,7 @@
 
 * Added *post_differentiation_time* and *post_differentiation_time_units* properties.
 * Added *perturbed* calculated property to indicate if the biosample has been perturbed using a treatment or a genetic modification for non-tagging purposes.
-* Added *post_nucleic_acid_delivery_time* and *post_nucleic_acid_delivery_time_units* properties.
+* Added *post_nucleic_acid_delivery_time* and *post_nucleic_acid_delivery_time_units* properties and to the calculated property of summary.
```

Comment:
Instead of expanding the line with "and to the .." please add a separate line, saying you have updated the summary property to include post nucleic acid delivery time info.

Reply:
Ok